tests/unit/io/test_io.py

```diff
@@ -494,11 +500,13 @@ def test_validate_and_regenerate_dataset(tmpdir):
     # Check that the regenerated dataset makes sense.
     # Dataset is ~544KiB - Expect 4 data files
     N = math.ceil(ddf.compute().memory_usage(deep=True).sum() / 150000)
-    file_list = glob.glob(os.path.join(path, "*"))
-    assert os.path.join(path, "_metadata") in file_list
-    assert os.path.join(path, "_file_list.txt") in file_list
-    assert os.path.join(path, "_metadata.json") in file_list
-    assert len(file_list) == N + 3  # N data files + 3 metadata files
+    data_file_list = glob.glob(os.path.join(path, "*"))
+    assert len(data_file_list) == N + 1  # N data files + '_metadata'
+    assert os.path.join(path, "_metadata") in data_file_list
+    metadata_file_list = glob.glob(os.path.join(path, ".merlin", "*"))
+    assert os.path.join(path, ".merlin", "_file_list.txt") in metadata_file_list
+    assert os.path.join(path, ".merlin", "_metadata.json") in metadata_file_list
```

Comment:
@rjzamora This PR looks good to me, but I wanted to run these files by you before we move them. Can these move to the `.merlin` directory without breaking things that depend on them?

Reply:
@karlhigley Thanks for catching this. I think these are metadata files for hugeCTR, and I was maybe thinking hugeCTR is also Merlin so... After double-checking some of hugeCTR code, I think moving those files might break hugeCTR. I reverted those changes in [a4c7cef](https://github.com/NVIDIA-Merlin/core/pull/249/commits/a4c7ceffe1a1d7473891b58e11236e4e3c804f06) to be safe so that these are still in the same directory as the parquet files, and only `schema.json` goes into `.merlin`.

Reply:
I agree with you that in the longer term they probably should get consolidated into the Merlin metadata directory, but this does seem safer for now 🤝